builder/bash/build-slugbuilder-slugrunner.bash

```diff
@@ -0,0 +1,41 @@
+set -eo pipefail
+
+# set debug based on envvar
+[[ $DEBUG ]] && set -x
+
+main() {
+  DOCKER="/usr/bin/docker"
+  DRIVER_OVERRIDE="$(cat /etc/docker.env)"
```

Comment:
I believe the best way to override defaults with an initialization script is to source `/etc/default/docker`. It may be better to write out a file with all file flags like so (similar to how the daemon is set up in Ubuntu):

    EXTRA_ARGS="--insecure-registry 10.0.0.0/8 --insecure-registry 172.16.0.0/12 --insecure-registry 192.168.0.0/16 --insecure-registry 100.64.0.0/10"